### PR TITLE
Jon/network switching

### PR DESCRIPTION
**Pull request by jonandgon**
## Description
This PR adds support for (future) network switching.

This is a companion PR to https://github.com/puzzlehq/firenze/pull/396

### CODE REVIEW

File: packages/react/src/store.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { create } from 'zustand';
 import { queryClient } from './index.js';
 import { persist } from 'zustand/middleware';
-import { PuzzleAccount } from '@puzzlehq/sdk-core';
+import { Network, PuzzleAccount } from '@puzzlehq/sdk-core';
```
Comment (lukenewman) [inline packages/react/src/store.ts:4]: used?

File: packages/react/src/index.ts [hunk 2]
```diff
@@ -16,6 +16,5 @@ export * from './hooks/wc/useOnSessionExpire.js';
 export * from './hooks/wc/useOnSessionUpdate.js';
 export * from './hooks/wc/useSession.js';
 export * from './provider/PuzzleWalletProvider.js';
-export * from './store.js';
```
Comment (lukenewman) [inline packages/react/src/index.ts:19]: can remove the file too?

**Review by lukenewman — APPROVED** [review]
few cleanup comments

File: packages/react/src/store.ts (hunk 1 above)
Comment (jonandgon) [reply packages/react/src/store.ts:4]: removed

File: packages/react/src/index.ts (hunk 2 above)
Comment (jonandgon) [reply packages/react/src/index.ts:19]: No, we need the wallet store, but we shouldn't export it from the index file. I don't want devs to directly access it

**Pull request merged**